packages/contracts/test/diamond/DiamondTestSetup.sol

```diff
@@ -404,6 +404,13 @@ abstract contract DiamondSetup is DiamondTestHelper {
 
         DiamondInit.Args memory initArgs = DiamondInit.Args({
             admin: admin,
+            /*
+            // cspell: disable-next-line
+            dollarName: "Ubiquity Algorithmic Dollar",
+            // cspell: disable-next-line
+            dollarSymbol: "uAD",
+            dollarDecimals: 18,
+            */
```

Comment:
@pavlovcik These threw an error on compile cause the `Args` struct in `DiamondInit` only contains 6 values. I commented them out rather than removing them cause it was unclear if they should be added to the struct or not.